### ISSUE TITLE
Barely a few KB of language files removed

### ISSUE THREAD

**Issue by Zetagonality** [IngmarStein/Monolingual#130]
Almost no language files were deleted. I suspect the reason is SIP, because I could not delete them manually even as root. Maybe I'm just doing something wrong?

**Comment by IngmarStein** [comment]
Were you running Monolingual for the first time? On most systems, you'll find quite a few language files not protected by SIP. Are the "few KB" reported by Monolingual or were you looking at Finder's numbers?

**Comment by Zetagonality** [comment]
Yes, I was running it for the first time. The few kilobytes were reported by Monolingual. Should I disable SIP and try again?

**Comment by IngmarStein** [comment]
That is unusual. Did you have a look at the log file do check if Monolingual had attempted to delete the files you expect it to remove?
Can you provide a screenshot of the main Monolingual screen (i.e. before starting the process)?

**Comment by Zetagonality** [comment]
Where is the log file located? I will provide screenshots in a day or so, when I'm less busy.

**Comment by IngmarStein** [comment]
No worries. The log is in ~/Library/Logs/Monolingual.log.

**Comment by Zetagonality** [comment]
Sorry for the late reply. Here are the screenshots of the main window, before clicking 'Remove': 
![screen shot 2018-03-16 at 7 00 20 pm](https://user-images.githubusercontent.com/32086664/37523281-79162bfe-294c-11e8-90bf-cf14b8cac311.png)
![screen shot 2018-03-16 at 7 00 43 pm](https://user-images.githubusercontent.com/32086664/37523283-794e740a-294c-11e8-8189-e27eee566b58.png)
And after running it, it reports that only one byte is saved!
![screen shot 2018-03-16 at 7 04 43 pm](https://user-images.githubusercontent.com/32086664/37523446-f7e7ee5e-294c-11e8-9b36-3e8214c3c1d5.png)
The log file :
[Monolingual.log](https://github.com/IngmarStein/Monolingual/files/1819350/Monolingual.log)

**Comment by Zetagonality** [comment]
At the end of the log, it shows that no files were deleted. Should I try disabling SIP before running it to see if that is the cause of the issue?

**Comment by marwanfayed** [comment]
Same problem here. Most recent log entry below. Everything is exactly as described above.

  2018-07-08T12:06:26Z Monolingual 1.8.0 started
  2018-07-08T12:06:26Z Removing languages: ach.lproj [...long list of languages...]
  2018-07-08T12:06:26Z Deleted files:

Thanks for all the hard work! Monolingual is a great project.
